pkg/controller/perconaservermongodbbackup/backup.go

```diff
@@ -149,6 +149,18 @@ func (b *Backup) Status(ctx context.Context, cr *api.PerconaServerMongoDBBackup)
 		status.CompletedAt = &metav1.Time{
 			Time: time.Unix(meta.LastTransitionTS, 0),
 		}
+
+		if cluster.Spec.Backup.PITR.Enabled {
+			tl, err := b.pbm.GetLatestTimelinePITR(ctx)
+			if err != nil && err != backup.ErrNoOplogsForPITR {
+				return status, errors.Wrap(err, "get latest PITR timeline")
+			}
+			if err == nil {
+				status.LatestRestorableTime = &metav1.Time{
```

Comment:
we need to update the latest backup object as soon as we have new chunk
```
  PITR chunks [77.36KB]:
    2024-03-22T09:39:06Z - 2024-03-22T09:41:15Z
    2024-03-22T09:34:44Z - 2024-03-22T09:38:12Z
```

in backup object we have old info
```
status:
  completed: "2024-03-22T09:39:08Z"
  destination: s3://operator-testing/2024-03-22T09:38:59Z
  lastTransition: "2024-03-22T09:39:08Z"
  latestRestorableTime: "2024-03-22T09:38:12Z"
```